jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+
+3rd party orbs are those published by our customers and other members of our community. For you to publish orbs or for your projects to use 3rd party orbs, your organization must opt-in under SECURITY within the Organization Settings page under the section "Orb Security Settings" where an organization administrator must agree to the terms for using 3rd-party software.
+
+<aside class="notice">
+This setting can only be toggled by organization administrators.
+</aside>
+
+### Orb reference format
+
+Orb references have the following format shown below:
+
+```
+[namespace]/[orb]@[version]
+```
+
+### Semantic Versioning in Orbs
+
+Orbs are published with the standard 3-number semantic versioning system:
+
+* major
+* minor
+* patch
+
+Orb authors need to adhere to semantic versioning. Within ```config.yml```, you may specify wildcard version ranges to resolve orbs. You may also use the special string ```volatile``` to pull in whatever the highest version number is at time your build runs.
+
+For example:
+
+* when ```mynamespace/some-orb@8.2.0``` exists, and ```mynamespace/some-orb@8.1.24``` and ```mynamespace/some-orb@8.0.56``` are published after 8.2.0, ```volatile``` will still refer to ```mynamespace/some-orb@8.2.0``` as the highest semantic version.
+
+Examples of orb version declarations and their meaning:
+
+1. ```circleci/python@volatile``` - use the latest version of the Python orb in the registry at the time a build is triggered.
+2. ```circleci/python@2``` - use the latest version of version 2.x.y of the Python orb .
+3. ```circleci/python@2.4``` - use the latest version of version 2.4.x of the Python orb .
+4. ```circleci/python@3.1.4``` - use exactly version 3.1.4 of the Python orb.
+
+#### Using Development Versions
+
+While all production orbs must be published securely by organization administrators, development orbs allow your team much broader latitude.
+
+A development version must be referenced entirely, such as: ```mynamespace/myorb@dev:mybranch.```; whereas production orbs allow wildcard semantic version references. Note that there are no shorthand conveniences for development versions.
+
+<aside class="notice">
+Dev versions are mutable and expire: their contents can change, and they are subject to deletion after 90 days; therefore, it is strongly recommended you do not rely on a development versions in any production software, and use them only while actively developing your orb.
+</aside>
+
+## Importing an Orb
+
+Importing a set of existing orbs is a simple process that only requires you to provide the following instruction in your configuration. In the example below, you can see how to import a CircleCI Slack and Heroku orb:
+
+```orbs:
+  slack: circleci/slack@0.1.0
+  heroku: circleci/heroku@volatile
+```
+
+### Designing Orbs
+
+When designing your own orbs, make sure your orbs meet the following requirements:
```

Comment:
Do we have a link to all of the different orb components (commands, executors, etc.) that can go in here? This section feels like a good intro to orb design, and it would be nice to link to the sort of "advanced resources" for super users.

Reply:
Can this section be in the same format as our config docs? These ones have been super useful to us even in designing orbs: https://circleci.com/docs/2.0/configuration-reference/

Reply:
added the link, not sure what you mean about the format

Reply:
https://circleci.atlassian.net/browse/CIRCLE-14649